mithril-common/Cargo.toml

```diff
@@ -22,6 +22,7 @@ async-trait = "0.1.73"
 bech32 = "0.9.1"
 blake2 = "0.10.6"
 chrono = { version = "0.4.31", features = ["serde"] }
+ckb-merkle-mountain-range = "0.6.0"
```

Comment:
I'd recommend downgrading this dependency to v0.5.2. I haven't had the time to look into the changes in v0.6.0 but given that we just received this very suspicious pr (https://github.com/polytope-labs/hyperbridge/pull/362) to upgrade to v0.6.0. I'd say it has been compromised somehow. 

We use v0.5.2 because it was audited by the polkadot-sdk team, which also uses v0.5.2 (https://github.com/paritytech/polkadot-sdk/blob/master/Cargo.toml#L866)